README.md

```diff
@@ -22,3 +22,9 @@ The full list of contributors for se-edu can be found [here](https://se-edu.gith
 * **Bug reports, Suggestions** : Post in our [issue tracker](https://github.com/se-edu/addressbook-level2/issues)
   if you noticed bugs or have suggestions on how to improve.
 * **Contributing** : We welcome pull requests. Follow the process described [here](https://github.com/oss-generic/process)
+
+#Changes
+* Encapsulate CommandResult class members.
+* Split Address into more classes.
+* Handle 'file readonly' situation.
+* Tests updated.
```

Comment:
This is not how you should update the README file. At least for your this week LOs, you should update the user guide and/or developer guide..